cpu/native/startup.c

```diff
@@ -314,8 +314,8 @@ void usage_exit(int status)
 #endif
 #if defined(MODULE_SOCKET_ZEP) && (SOCKET_ZEP_MAX > 0)
 "    -z [<laddr>:<lport>,]<raddr>:<rport> --zep=[<laddr>:<lport>,]<raddr>:<rport>\n"
-"        provide a ZEP interface with local address and port (<laddr>, <lport>)\n"
-"        and remote address and port (default local: [::]:17754).\n"
+"        provide a ZEP interface with (optional) local address and port (<laddr>, <lport>).\n"
+"        The ZEP interface will connect to the remote address and optionally listen on a local address.\n"
```

Comment:
Line-length...

Reply:
Patch changed, but lines are still longer than 80 chars and can be divided up ;-).

Reply:
I though 80 characters was only a recommendation…

Reply:
Only if you can't find a way around it, is my interpretation of that convention. Let's not make it worse, especially in terminal output. Right now, the output is broken on a small (size as opened by Gnome) terminal window (IIRC the other broken output about GPIO is also yours ;-)):

![image](https://user-images.githubusercontent.com/675644/97575251-c1d6fb00-19ec-11eb-8d9c-48cf21d91952.png)

Reply:
> (The other lines in this file are also > 80 chars, just right above this line)

They are in a huge minority btw. Disregarding this addition there are

https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L310-L314 (should be fixed)

https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L417 (a link, so no way around it)

The remaining are only overlapping a few characters. Could be fixed, but only if you are bone-headed about it:

https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L420-L421
https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L433
https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L455
https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L501
https://github.com/RIOT-OS/RIOT/blob/870c8920bd1b9125f2807c58a5e963fb4b0c6bce/cpu/native/startup.c#L608

Reply:
Ok, should fit now.